src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -203,12 +206,14 @@ protected ResponseEntity<Object> retrieveAllOrganisationOrById(String organisati
             //Received request to retrieve organisation with status
 
             organisationResponse = organisationService.findByOrganisationStatus(orgStatus.toUpperCase(), pageable);
+            totalRecords = ((OrganisationsDetailResponse) organisationResponse).getOrganisations().size();
```

Comment:
I think you need to get the total records present in the table, not the response returned from the pageable
For e.g. if there are 100 records in the organisation and the pageable is configured to get 20 records then the total records should be 100 instead of 20.
I think this code will give 20 instead of 100
the UI will create pages based on the total records in the table which is 100. Please correct me if I am wrong